OpenRA.Mods.RA/Traits/Infiltration/Infiltrates.cs

```diff
@@ -58,6 +58,10 @@ bool IsValidOrder(Actor self, Order order)
 			if (order.ExtraData == 0 && order.TargetActor == null)
 				return false;
 
+			// Should not infiltrate own or allied
+			if (order.TargetActor.Owner.IsAlliedWith(self.Owner))
```

Comment:
I guess he means a bool in the trait info which I don't think we need as setting it allow this only enables exploits.